telemetry/main_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package telemetry
+
+import (
+	"testing"
+
+	"github.com/pingcap/tidb/util/testbridge"
+	"go.uber.org/goleak"
+)
+
+var (
+	GetFeatureUsage = getFeatureUsage
```

Comment:
I don't know what you mean well. Maybe show a patch?

Reply:
like: 

```
// FIXME: EXPORT_TRICK ...
GetFeatureUsage = getFeatureUsage
```

My suggestion is to have tests splitted in two packages, one with the same package name as the code(no access problem), another is suffixed with `_test`(can only make use of exported functions). `EXPORT_TRICK` will be excellent when we want to search for all such tricks. Though it can also be archived by some scripts/source code parsing, we will look through all tests anyway.

Reply:
For these two line, I tend to write the actual test under `telemetry`, but it failed with cyclic deps.

Reply:
I get your point. You can create an issue to continue the discussion anyway. It provides a best practice for testing which may need a bit more time to see whether or not we really need it.

Reply:
Yeah, for now, a mark is enough. Not really necessary though...